pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/Namespaces.java

```diff
@@ -1361,4 +1361,39 @@ SchemaAutoUpdateCompatibilityStrategy getSchemaAutoUpdateCompatibilityStrategy(S
     void setSchemaAutoUpdateCompatibilityStrategy(String namespace,
                                                   SchemaAutoUpdateCompatibilityStrategy strategy)
             throws PulsarAdminException;
+
+    /**
+     * Get schema validation enforced for namespace.
+     * @param namespace
+     * @param
+     * @return the schema validation enforced flag
+     * @throws NotAuthorizedException
+     *             Don't have admin permission
+     * @throws NotFoundException
+     *             Namespace does not exist
+     * @throws PulsarAdminException
+     *             Unexpected error
+     */
+
+    boolean getSchemaValidationEnforced(String namespace)
+        throws PulsarAdminException;
+    /**
+     * Set schema validation enforced for namespace.
+     * if a producer without a schema attempts to produce to a topic with schema in this the namespace, the
+     * producer will be failed to connect. PLEASE be carefully on using this, since non-java clients don't
+     * support schema. if you enable this setting, it will cause non-java clients failed to produce.
+     *
+     * @param namespace
+     * @param enable
+     * @return
+     * @throws NotAuthorizedException
```

Comment:
Can you double check if the exceptions documented here are aligned with the error codes returned at server?